contrib/gpg.rc

```diff
@@ -1,10 +1,18 @@
 # -*-muttrc-*-
 #
-# Command formats for gpg.
+# Command formats for GnuPG
 #
-# This version uses gpg-2comp from
+# Version notes:
+#
+#   GPG 2.1 introduces the option "--pinentry-mode", which requires
+#   the "loopback" argument in instances where "--passphrase-fd" is
+#   used.
+#
+# The gpg-2comp found in some comments comes from
 #   http://70t.de/download/gpg-2comp.tar.gz
 #
+# $Id: gpg.rc,v 1.12 2001/12/11 09:33:57 roessler Exp $
+#
```

Comment:
Where does this come from?

Reply:
I forgot that I did that.  Those are anchors.  Some projects want comments to appear above the code they refer to, and some want comments to appear below the referenced code.  I've also worked on a project that had it both ways, depending on what kind of code it was.  The purpose of the anchors are to clearly attach the comment to the SLOC it refers to.